app/components/polls/polls-edit/polls-edit.html

```diff
@@ -0,0 +1,97 @@
+<top-bar
+logo-href="/cms/app/polls/"
+save-function="saveModel"
+save-disable-when="model.$dirty().length === 0 && !isNew">
+</top-bar>
+
+<div class='row poll-name'>
+  <div class='col-md-4'></div>
+  <div class="form-group col-md-4">
+    <label for="pollTitle">Poll Name</label>
+    <input id="pollTitle"
+    type="text"
+    ng-model="model.title"
+    class="form-control">
+  </div>
+</div>
+
+<div class='row poll-end-date'>
+  <div class='col-md-4'></div>
+  <div class="form-group col-md-4">
+    <label for="pollEndDate">Poll End Date</label>
+    <div class="input-group">
+      <input
+      id="pollEndDate"
+      type="text"
+      value="{{ model.closeDate.format('MM/DD/YY h:mm:ss a') }}"
+      class="form-control"
+      readonly>
+      <span class="input-group-btn">
+        <button
+          class="btn btn-info"
+          datetime-selection-modal-opener
+          ng-model="model.closeDate"
+          modal-title="Poll End Date">
+          <i class="glyphicon glyphicon-calendar"></i>
```

Comment:
Use `font-awesome` instead of `glyphicons`, this will make merges into `bulbs-cms-2` easier.